runtime/runtime-params-estimator/src/estimator_context.rs

```diff
@@ -406,13 +413,28 @@ impl Testbed<'_> {
     /// Returns the number of blocks required to reach quiescence
     fn process_blocks_until_no_receipts(&mut self, allow_failures: bool) -> usize {
         let mut n = 0;
-        while !self.prev_receipts.is_empty() {
+        while self.has_unprocessed_receipts() {
             self.process_block_impl(&[], allow_failures);
             n += 1;
         }
         n
     }
 
+    fn has_unprocessed_receipts(&self) -> bool {
+        if !self.prev_receipts.is_empty() {
+            return true;
+        }
+
+        // Check congestion info to see if there are any queued receipts.
+        for (_, extended_congestion_info) in self.apply_state.congestion_info.iter() {
+            if extended_congestion_info.congestion_info.receipt_bytes() > 0 {
```

Comment:
Just to make sure I understand correctly, is that condition changed to also look at buffered receipts?